app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -247,10 +264,15 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       ($scope.emsCommonModel.default_hostname != '' && $scope.emsCommonModel.default_api_port) &&
       ($scope.emsCommonModel.default_password != '' && $scope.angularForm.default_password.$valid)) {
       return true;
-    } else if(($scope.emsCommonModel.metrics_selection != "disabled" && $scope.emsCommonModel.ems_controller == "ems_container") &&
+    } else if(($scope.emsCommonModel.ems_controller == "ems_container") &&
```

Comment:
Use `===` instead of `==`

Also, a space is preferred after `if` per style guide

Reply:
@AparnaKarve Thanks 👍 the `if` with no space, looked wrong ... but codeclimate didn't shout so I thought is was ok :-)